src/test/java/tests.java

```diff
@@ -0,0 +1,135 @@
+import org.json.JSONException;
+import org.junit.Before;
+import org.junit.Test;
+import srver.PO;
+import srver.server;
+import srver.vulnerability;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static srver.server.readJsonFromUrl;
+
+
+/**
+ * Created by Ксения on 12.12.2016.
+ */
+
+public class tests {
+    public vulnerability vuln;
+
+
+
+
+    @Before
+    public void before() {
+        server s = new server();
```

Comment:
&#x1F34E; **Issue**
You should use more descriptive names than "s". 10 lines below you won't even remember, what "s" actually means, while, name like "server" you won't forget.